config/routes.rb

```diff
@@ -10,10 +10,11 @@
   post '/cart_items', to: "cart_items#create"
   get '/cart_items', to: "cart_items#show", as: "cart"
   post '/order_cats', to: "order_cats#create"
+  delete '/cart_items', to: "cart_items#delete"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.